connector/connector.go

```diff
@@ -65,18 +65,24 @@ type CallbackConnector interface {
 	HandleCallback(s Scopes, r *http.Request) (identity Identity, err error)
 }
 
+// SAMLConnector bindings
+const (
+	SAMLBindingPOST     = "post"
```

Comment:
nit: this should probably be httpPost and httpRedirect, right? Since the other ones are SOAP based.